server/src/main/java/io/pravega/schemaregistry/storage/impl/group/InMemoryGroupTable.java

```diff
@@ -86,7 +86,7 @@ public CompletableFuture<Void> updateEntries(List<Entry<Integer>> updates) {
             TableKey key = update.getKey();
             Integer version = update.getVersion();
             Value<TableValue, Integer> val = table.get(key);
-            return version == null || (val != null && version.equals(val.getVersion()));
+            return version == null || val != null && version.equals(val.getVersion());
```

Comment:
Didn't get the reason behind this change

Reply:
As we are changing the Checkstyle version it was claimed by that, it was claiming that `redundant Pair of parenthesis`